test/int/databaseuser_unprotected_test.go

```diff
@@ -375,11 +375,6 @@ var _ = Describe("Atlas Database User", Label("int", "AtlasDatabaseUser", "prote
 				// We need to make sure that the new connection secret is different from the initial one
 				connSecretUpdated := validateSecret(k8sClient, *testProject, *testDeployment, *testDBUser1)
 				Expect(string(connSecretUpdated.Data["password"])).To(Equal("someNewPassw00rd"))
-
-				var updatedPwdSecret corev1.Secret
-				Expect(k8sClient.Get(context.Background(), kube.ObjectKey(testNamespace.Name, UserPasswordSecret), &updatedPwdSecret)).To(Succeed())
-				Expect(testDBUser1.Status.PasswordVersion).To(Equal(updatedPwdSecret.ResourceVersion))
```

Comment:
Note: this assertion is wrong in the first place. There is no guarantee we will ever see the resource version the operator reconciled on the ginkgo client. This already validates the updated secret two lines above so removing this is just removing a redundant assertion.